### PR TITLE
Add SlurmGCP v6 example of htc-slurm blueprint

### PR DESCRIPTION
**Pull request by harshthakkar01**
### Submission Checklist

Please take the following actions before submitting this pull request.

* Fork your PR branch from the Toolkit "develop" branch (not main)
* Test all changes with pre-commit in a local branch [#](https://goo.gle/hpc-toolkit#development)
* Confirm that "make tests" passes all tests
* Add or modify unit tests to cover code changes
* Ensure that unit test coverage remains above 80%
* Update all applicable documentation
* Follow Cloud HPC Toolkit Contribution guidelines [#](https://goo.gle/hpc-toolkit-contributing)

### CODE REVIEW

File: community/modules/scheduler/schedmd-slurm-gcp-v6-controller/etc/htc-slurm.conf.tpl [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+# https://slurm.schedmd.com/high_throughput.html
+
+ProctrackType=proctrack/cgroup
+SlurmctldPidFile=/var/run/slurm/slurmctld.pid
+SlurmdPidFile=/var/run/slurm/slurmd.pid
+TaskPlugin=task/affinity,task/cgroup
+MaxArraySize=10001
+MaxJobCount=500000
+MaxNodeCount=100000
+MinJobAge=60
```
Comment (nick-stroud) [inline community/modules/scheduler/schedmd-slurm-gcp-v6-controller/etc/htc-slurm.conf.tpl:11]: Can you confirm that these actually make it into the deployed cluster. I believe they would be under /etc/slurm/slurm.conf. Same for db settings. For example, you should see `MaxArraySize=10001` in the deployed cluster.

**Review by nick-stroud — APPROVED** [review]

File: community/examples/htc-slurm-v6.yaml
Comment (nick-stroud) [inline community/examples/htc-slurm-v6.yaml:1]: Shouldn't there be a README change that goes along with this?

File: community/modules/scheduler/schedmd-slurm-gcp-v6-controller/etc/htc-slurm.conf.tpl (hunk 1 above)
Comment (harshthakkar01) [reply community/modules/scheduler/schedmd-slurm-gcp-v6-controller/etc/htc-slurm.conf.tpl:11]:
```
[root@htcslurmv6-controller slurm]# cat slurm.conf 
# slurm.conf
# https://slurm.schedmd.com/high_throughput.html

ProctrackType=proctrack/cgroup
SlurmctldPidFile=/var/run/slurm/slurmctld.pid
SlurmdPidFile=/var/run/slurm/slurmd.pid
TaskPlugin=task/affinity,task/cgroup
MaxArraySize=10001
MaxJobCount=500000
MaxNodeCount=100000
MinJobAge=60
```

Comment (harshthakkar01) [reply community/modules/scheduler/schedmd-slurm-gcp-v6-controller/etc/htc-slurm.conf.tpl:11]: For slurmdbd
```
root@htcslurmv6-controller slurm]# cat slurmdbd.conf 
# slurmdbd.conf
# https://slurm.schedmd.com/slurmdbd.conf.html

DebugLevel=info
PidFile=/var/run/slurm/slurmdbd.pid

# https://slurm.schedmd.com/slurmdbd.conf.html#OPT_CommitDelay
CommitDelay=1

################################################################################
#              vvvvv  WARNING: DO NOT MODIFY SECTION BELOW  vvvvv              #
################################################################################

AuthType=auth/munge
AuthAltTypes=auth/jwt
AuthAltParameters=jwt_key=/var/spool/slurm/jwt_hs256.key

DbdHost=htcslurmv6-controller

LogFile=/var/log/slurm/slurmdbd.log

SlurmUser=slurm

StorageLoc=slurm_acct_db

StorageType=accounting_storage/mysql
StorageHost=localhost
StoragePort=3306
StorageUser=slurm
StoragePass=""

################################################################################
#              ^^^^^  WARNING: DO NOT MODIFY SECTION ABOVE  ^^^^^              #
################################################################################
```

File: community/examples/htc-slurm-v6.yaml
Comment (harshthakkar01) [reply community/examples/htc-slurm-v6.yaml:1]: Thanks for catching. Updated.

**Review by nick-stroud — APPROVED** [review]

**Pull request merged**